src/current/v23.1/monitoring-and-alerting.md

```diff
@@ -189,6 +189,685 @@ If you rely on external tools for storing and visualizing your cluster's time-se
 
 When storage of time-series metrics is disabled, the DB Console Metrics dashboards in the DB Console are still available, but their visualizations are blank. This is because the dashboards rely on data that is no longer available.
 
+### Critical nodes endpoint
+
+The critical nodes status endpoint is used to:
+
+- Check if any of your nodes are critical.  A node is 'critical' if that node becoming unreachable would cause [replicas to become unavailable](ui-cluster-overview-page.html#replication-status).
+- Check if any ranges are [under-replicated or unavailable](ui-cluster-overview-page.html#replication-status). This is useful when determining whether a node is ready for [decommissioning](node-shutdown.html#decommissioning).
+- Check if any of your cluster's data placement constraints (set via [Multiregion SQL](multiregion-overview.html) or direct [configuration of replication zones](configure-replication-zones.html)) are being violated. This is useful when implementing [data domiciling](data-domiciling.html).
```

Comment:
```suggestion
- Check if any of your cluster's data placement constraints (set via [multi-region SQL](multiregion-overview.html) or direct [configuration of replication zones](configure-replication-zones.html)) are being violated. This is useful when implementing [data domiciling](data-domiciling.html).
```

Reply:
I'm trying to make ~fetch~ "multiregion" happen but this is ok! updated! :-)